core/app/models/spree/order_updater.rb

```diff
@@ -198,21 +198,8 @@ def update_promotions
       Spree::Config.promotions.order_adjuster_class.new(order).call
     end
 
-    def update_taxes
```

Comment:
Did you consider keeping this method with just a deprecation inside? 

```ruby
def update_taxes
  Spree::Deprecation...
  update_tax_adjustments
end
```

This way, people overriding this method (only when using `super`) will be notified and can easily change to the new version. I know it's private, but it's also a target for many customizations.

By the way, this is not a blocker, just wanted to hear your opinion on some extra safe check, which I can definitely live without.

Reply:
Do you know of stores or extensions that override it? I can check with my team as well.

Reply:
I don't know about you three but Avatax has barely any downloads on rubygem and hasn't been updated in 7 years. I would give this a nudge on the merge button and use the new code to make some bugfixing around calculations that are overdue.

Reply:
> I'm not sure the deprecation really works in the case of overrides, actually. People who override it won't see the deprecation. Their override will just stop doing anything.

@jarednorman  it would trigger if the override calls `super` into it. By the way, merging this as is was a good call. It's a private method and we should be less conservative in favor of progress.